style.css

```diff
@@ -0,0 +1,947 @@
+  background: #fff;
+  border-top-right-radius: 100px;
+}
+
+.myself {
+  width: 264px;
+  height: 101px;
+  display: flex;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  flex: none;
+  margin-top: 114px;
+  margin-left: 24px;
+  margin-right: 24px;
+  margin-bottom: 15px;
+}
+
+#paragraph {
+  margin-left: 24px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  width: 327px;
+  height: 144px;
+  margin-bottom: 20px;
+  color: #344563;
+}
+
+.get {
+  width: 170px;
+  height: 48px;
+  padding: 12px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  margin-top: 30px;
+  margin-left: 24px;
+}
+
+.resume {
+  width: 138px;
+  height: 24px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #396df2;
+}
+
+.languages {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-left: 36px;
+  margin-right: 45px;
+  margin-top: 30px;
+}
+
+.lang {
+  height: 24px;
+  width: 237px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  color: #000;
+}
+
+#list {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.list {
+  display: flex;
+  justify-content: flex-start;
+  gap: 20px;
+  width: 305px;
+  height: 67px;
+  margin-top: 20px;
+  margin-left: 0;
+  background: #f7f7f9;
+  border-radius: 8px;
+}
+
+.lang-image {
+  margin-left: 10px;
+}
+
+.lang-list {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #253858;
+}
+
+#lang-list {
+  margin-bottom: 50px;
+}
+
+.frameworks {
+  display: flex;
+  align-items: center;
+  margin-left: 38px;
+  justify-content: space-between;
+  margin-right: 45px;
+}
+
+.line {
+  margin: 26px 15px;
+  background: #dfe1e6;
+  width: 327px;
+  height: 2px;
+}
+
+.skills {
+  width: 237px;
+  height: 24px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  color: #000;
+}
+
+.get:hover {
+  background-color: #396df2;
+  transition-duration: 0.05s;
+  cursor: pointer;
+}
+
+.resume:hover {
+  color: #fff;
+  transition-duration: 0.05s;
+  font-weight: bold;
+  width: auto;
+}
+
+/* contact form */
+.contact-me {
+  width: 375px;
+  height: 790px;
+  background: #6070ff;
+  border-top-left-radius: 100px;
+  margin-top: -160px;
+  background-image: url(./images/contact-shapes.png);
+  background-repeat: no-repeat;
+  background-position-y: 9px;
+  background-position-x: 153px;
+}
+
+#contact-me {
+  width: auto;
+  height: 56px;
+  text-align: center;
+  margin-top: 78px;
```

Comment:
Kindly, I recommend you to use flexbox to adapt the information to the center of the container